XCUITests/FindInPageTest.swift

```diff
@@ -13,6 +13,16 @@ class FindInPageTests: BaseTestCase {
         XCTAssertTrue(app.textFields[""].exists)
     }
 
+    func testFindInLargeDoc() {
+        userState.url = "http://localhost:6571/find-in-page-test.html"
+        openFindInPageFromMenu()
+
+        // Enter some text to start finding
+        app.textFields[""].typeText("Book")
+        waitforExistence(app.staticTexts["1/500+"])
```

Comment:
The test is failing for me here. After typing "Book" it finds 1/14000, then it quickly changes to 1/3000 and finally to 1/2000. If we wait for  "1/2000" instead of  "1/500+" the test works for both iPhone and iPad sim.

Reply:
Oops, I didn't merge master into this branch, on master, 500 is the highest number in the UI

Reply:
Actually BB will do the merge-to-master when running tests, and two unrelated ones failed:
https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5abaaf088fa911000112dbd8#tests
Running again anyway to see if I get a green build so that the Merge button on github becomes active.

Reply:
Oh I see, yeah, now with latest master on this branch I can si that number. If BB is green, then great! Those two failures are known intermittents, there is a fix for the first one at least ready to land.